canonical-web-and-design/jaas-dashboard
# Use npm react components

## Done

- Remove MainTable components.
- Use MainTable from @canonical/react-components.
- Mock imported scss files.

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8036/
- Check that the model list and model details pages appear exactly as they did before.
